app/scripts/app.ts

```diff
@@ -15,6 +16,10 @@ import { ECS_MODULE } from '@spinnaker/ecs';
 import '@spinnaker/cloudfoundry';
 import { AZURE_MODULE } from '@spinnaker/azure';
 
+initPlugins().catch(() => {
+  //TODO use CustomBanner to tell the user that plugin(s) have not been loaded
```

Comment:
With this method, you could potentially, start the app before plugins are loaded. This might be ok for stages but that limits the ability of plugins to affect the lifecycle of the app.

To avoid it, `initPlugins().catch(notifytheuser).finally(starttheapp)`